distribution/docs/src/main/resources/_contents/_solr-contents/integrating-solr-contents.adoc

```diff
@@ -11,6 +11,7 @@ Some notable features of the Solr Catalog Provider include:
 ** No configuration necessary on a single-node distribution
 ** Data directory of Solr indexes are configurable
 * Works with a standalone Solr Server
+* Works with Solr Cloud
```

Comment:
I updated [PR 1419, DDF-2400](https://github.com/codice/ddf/pull/1419) to match this.